SessionUtilitiesKit/Crypto/Mnemonic.swift

```diff
@@ -118,7 +118,9 @@ public enum Mnemonic {
     }
     
     public static func decode(mnemonic: String, language: Language = .english) throws -> String {
-        var words: [String] = mnemonic.components(separatedBy: .whitespacesAndNewlines)
+        var words: [String] = mnemonic
+            .components(separatedBy: .whitespacesAndNewlines)
+            .map { $0.trimmingCharacters(in: .whitespacesAndNewlines) }
```

Comment:
Hmmm... Do we have to add this trimming? This doesn't seem to do anything. :thinking:

Reply:
And we may want to add `.filter { !$0.isEmpty() }`

Reply:
Yea not sure what I was trying to do with the trimming 🤔 

Good catch with the `filter` - have added